department-of-veterans-affairs/vets-website
# 47468 national cemetary note aria

## Description
This work programmatically links the national cemeteries note to the cemetery question using `aria-describedby`

## Original issue(s)
department-of-veterans-affairs/va.gov-team#47468


## Testing done
- MacOS/Chrome - inspector
- MacOS/Chrome + VoiceOver

## Screenshots
<img width="730" alt="Screen Shot 2022-10-06 at 5 59 44 PM" src="https://user-images.githubusercontent.com/1094951/194426580-a3e0a572-baf6-4d72-b3c0-7db1c43d1e09.png">
<img width="367" alt="Screen Shot 2022-10-06 at 6 00 42 PM" src="https://user-images.githubusercontent.com/1094951/194426581-9b2f6beb-a2cf-48f1-91d4-309bf3715d0a.png">

## How to test

1. [Begin the pre need flow unsigned in ](https://localhost:3001/burials-and-memorials/pre-need/form-10007-apply-for-eligibility/applicant-information)
2. Fill out the form to get to step 3
3. Turn on your screen reader and tab through the page to focus on the field named "Which VA national cemetery would you prefer to be buried in?"
4. Confirm that the note "Please note: This doesn’t guarantee you’ll be buried in your preferred cemetery. We’ll try to fulfill your wishes, but will assign a gravesite in a cemetery with available space at the time of need." is read out loud after the name of the input. 

... or alternatively

3a: Run the web inpsector on the input for "Which VA national cemetery would you prefer to be buried in?"
4a: Confirm the . 
5b


## Acceptance criteria
- [x] Tests continue to pass
- [x] In Voice Over, the note is read by Voice
- [x] Th input "Which VA national cemetery would you prefer to be buried in?" has the attribute `aria-describedby="burial-cemetary-note"`
- [x] The note has an `id` of  `burial-cemetary-note`.

## Definition of done
